cmd/segment-reaper/delete.go

```diff
@@ -0,0 +1,47 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/zeebo/errs"
+	"go.uber.org/zap"
+
+	"storj.io/storj/pkg/process"
+	"storj.io/storj/satellite/metainfo"
+)
+
+var (
+	deleteCmd = &cobra.Command{
+		Use:   "delete",
+		Short: "Deletes zombie segments from DB",
+		Args:  cobra.OnlyValidArgs,
+		RunE:  cmdDelete,
+	}
+
+	deleteCfg struct {
+		DatabaseURL string `help:"the database connection string to use" default:"postgres://"`
```

Comment:
The default won't work, why not just use `""`?

Reply:
hmm I have no strong opinion about it but with `postgres://` user at least know what kind of DB is supported. But I can set to `""` if you think it will be cleaner.

Reply:
no, I think that you have a good point.